docs/REST API/rest-api.md

```diff
@@ -8805,6 +8806,42 @@ echo "hello"
 }
 ```
 
+### Modify script
+
+Modifies an existing script.
+
+`PATCH /api/v1/fleet/scripts/:id`
+
+##### Request body
+
+```json
+{
+  "script_content": "#!/bin/sh\\n\\n#!/usr/bin/env bash\\n\\nsudo systemsetup -settimezone Pacific/Ponape"
+}
+```
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                           |
+| ----            | ------- | ---- | --------------------------------------------          |
+| id              | integer | path | **Required**. The ID of the script to modify. |
+| script_contents | string  | body | The contents of the script to run. Only one of either `script_id` or `script_contents` can be included in the request; omit this parameter if using `script_id`. |
+
+#### Example
+
+`PATCH /api/v1/fleet/scripts/1`
+
+##### Default response
+
+`Status: 204`
```

Comment:
`200` status code when there's response data. (`204` means "No content", so we only use it when there's no data sent back.)

```suggestion
`Status: 200`
```